pkg/storage/bloom/v1/bloom_tester.go

```diff
@@ -120,6 +191,33 @@ func (b stringTest) MatchesWithPrefixBuf(bloom filter.Checker, buf []byte, prefi
 	return true
 }
 
+type stringMatcherFilter struct {
+	test stringTest
+}
+
+// Filter implements the log.Filterer interface
+func (b stringMatcherFilter) Filter(_ []byte) bool {
+	panic("implement me")
+}
+
+// ToStage implements the log.Filterer interface
+func (b stringMatcherFilter) ToStage() log.Stage {
+	panic("implement me")
+}
```

Comment:
The simplifier needs to be able to return both types `Matcher` and `Filterer` so it needs to know how to create a type that implements both. In this case, since we will only be using the `Matcher` path, I think it's safe to just satisfy the `Filterer` interface with these panic impl.